internal/pkg/cli/job_deploy.go

```diff
@@ -159,6 +159,15 @@ func (o *deployJobOpts) Execute() error {
 	if err != nil {
 		return fmt.Errorf("upload deploy resources for job %s: %w", o.name, err)
 	}
+	serviceInRegion, err := deployer.IsServiceAvailableInRegion(o.targetEnv.Region)
+	if err != nil {
+		return fmt.Errorf("check if Scheduled Job(s) is available in region %s: %w", o.targetEnv.Region, err)
+	}
+
+	if !serviceInRegion {
+		log.Warningf(`Scheduled Job(s) might not be available in region %s; proceed with caution.
+`, o.targetEnv.Region)
+	}
```

Comment:
What do you think of checking service availability before `deployer.UploadArtifacts`? This way we error out much quicker.